lib/vector/vedit/vertex.c

```diff
@@ -157,16 +156,16 @@ int Vedit_move_vertex(struct Map_info *Map, struct Map_info **BgMap,
 
         if (rewrite) {
             if (Vect_rewrite_line(Map, line, type, Points, Cats) < 0) {
+                G_free(moved);
```

Comment:
Also have to destroy Points and Cats?
```suggestion
                Vect_destroy_line_struct(Points);
                Vect_destroy_cats_struct(Cats);
                G_free(moved);
```